### DISCUSSION TITLE
How does remote cache work in CI/CD

### DISCUSSION THREAD

**Discussion by samchouse** [vercel/turborepo#400] [Help] [answered]
How does remote caching work on CI/CD since you don't push your token to VSC?

**Comment by weyert** [comment, marked as answer]
You can pass the `TURBO_TOKEN ` environment variable, see: https://github.com/vercel/turborepo/blob/ac2793a0c976ff26bb0375c789453d37c91e5e28/docs/pages/docs/reference/command-line-reference.mdx#--token
